cpp/dolfinx/fem/utils.cpp

```diff
@@ -215,42 +206,6 @@ fem::compute_integration_domains(fem::IntegralType integral_type,
           "Cannot compute integration domains. Integral type not supported.");
     }
   }
-
-  // Build permutation that sorts by meshtag value
```

Comment:
It doesn't look like any sort is happening in `forms.py`, which is fine as long as the integration data isn't assumed to be sorted anywhere. Are we sure that there are no assumptions that integration data should be sorted by meshtag values?

Reply:
I’ve never seen such an assumption in the code. All tests pass, so i dont think it matters

Reply:
I've added in a sort for good measure